index.md

```diff
@@ -76,7 +76,7 @@ We provide consultancy and engineering in research projects in various formats:
 - We also offer **short-term engineering** support as a free service. During a period of a few months, you will have regular meetings with one or two Research Engineers, with the aim of developing a proof-of-concept or exploring a new method for your research.
```

Comment:
```suggestion
- We also offer **short-term engineering** support as a free service. During a period of a few months, you will work together with two or three Research Engineers, with the aim of developing a proof-of-concept or exploring a new method for your research.
```

Reply:
I am in favor of keeping the "have regular meetings", since it is more specific. Agree with the other change of course.

Reply:
How about "work together and have regular meetings"? I think only "have regular meetings" may be suggesting there will be only meetings

Reply:
Sounds good to me!